group-generators/generators/optimists/index.ts

```diff
@@ -0,0 +1,34 @@
+import { AttestationStationProvider } from "@group-generators/helpers/data-providers/atst";
+import { ValueType, Tags, GroupWithData } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Weekly,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const provider = new AttestationStationProvider();
+    const receivers = await provider.getAttestations(
```

Comment:
You need to create a type that contains the key and the value rather than pass the 2 values in 2 diferents arguments.

Because when the Factory generate the group, it will tke the argument and create an object from it, so like this:
`const receivers = await provider.getAttestations({key,value})`

So you will have to create a getAttestations type like this:
`export type getAttestations={attestation:string}`

You can inspire you from this data provider if you want: https://github.com/sismo-core/sismo-hub/tree/main/group-generators/helpers/data-providers/talentlayer